packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -372,6 +373,12 @@ class DynamicSidecarServiceLabels(BaseModel):
         description="allow complete internet access to containers in here",
     )
 
+    callbacks_mapping: Json[CallbacksMapping] | None = Field(
```

Comment:
would the code not be simpler if you go with something like making it *not* optional but with a default of an empty dict?